.github/workflows/R-CMD-check.yaml

```diff
@@ -57,6 +49,7 @@ jobs:
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
       R_KEEP_PKG_SOURCE: yes
+      _R_CHECK_CRAN_INCOMING_REMOTE_: false
       _R_CHECK_CRAN_INCOMING_: false
       _R_CHECK_FORCE_SUGGESTS_: false
```

Comment:
@etiennebacher This env var makes sure that there won't be a note if a suggested package is missing.

Reply:
Thanks for the explanation